app/controllers/system/admin/instance/controller.rb

```diff
@@ -12,7 +12,7 @@ def load_instance
   end
 
   def authorize_instance_admin
-    authorize!(:manage, @instance)
+    authorize!(:show, @instance)
```

Comment:
Problem with this is that everyone can see the instance, so this doesn't check anything...

Maybe for instances we should have another verb, maybe `:modify`?